### DISCUSSION TITLE
MG 1.8.1 not talking to HWInfo64 Pro v7.15-4620

### DISCUSSION THREAD

**Discussion by sdholden** [raiguard/ModernGadgets#271] [Q&A] [answered]
Just updated MG to 1.8.1 and triple-checked the settings in latest HWInfo as per the tutorial link from MG1.8.1.  Just get "ERROR:  HWiNFO is not configured or is not running.", even after a reboot and 4th config verification check.  I set some index values fromm the HWInfo sensor page itself, since it also shows the indexes, and refreshed all gadgets.  Still no change to gadgets and sensors list shows error.  Any other hints or suggestions to try?

![HWI Settings](https://user-images.githubusercontent.com/14812533/145327217-30e3ed7f-4e1e-427b-a170-78ffd12b0944.png)
![HWI Sensors](https://user-images.githubusercontent.com/14812533/145327307-20001399-835f-483c-882a-46a96fbf919a.png)
?

**Comment by SilverAzide** [comment]
Based on your screenshots, you did the first step to configure HWiNFO.  Did you do the final step to configure the actual ModernGadgets skins?
https://github.com/raiguard/ModernGadgets/wiki/HWiNFO-Configuration#configuring-moderngadgets

If so, post your HWiNFOIndexList.txt and HWiNFOSettings.inc files.

**Comment by sdholden** [comment]
Yes, I'll post them in the morning.  
The HWiNFOIndexList.txt has only the error message.  I therefore used the index numbers from the HW Sensors config dialog to insert its index #'s into the .inc file...as a test.

Thanks for the quick reply!

**Comment by sdholden** [comment]
Here's the two files..I added .txt to the .inc file to allow pasting here...
[HWiNFOIndexList.txt](https://github.com/raiguard/ModernGadgets/files/7685141/HWiNFOIndexList.txt)
[HWiNFOSetting.inc.txt](https://github.com/raiguard/ModernGadgets/files/7685154/HWiNFOSetting.inc.txt)
.

**Comment by SilverAzide** [comment, marked as answer]
Any chance you are not an administrator on your machine?  If you run Rainmeter as an administrator as a test, do the skins work?

**Comment by sdholden** [comment]
My primary account is a std user, with a separate admin account.  Habit from old windows days to ensure viruses have a harder time to get in. Will try Rainmeter as admin to see...easy enough to do that all the time, if it works!

Thanks for the idea!

⁣Get BlueMail for Android ​

**Reply by sdholden** [reply]
Yes, running Rainmeter as Admin allows MG to see the HWiNFO data. New wrinkle for v7 of HWiNFO, as 6.x didn't exhibit this issue.

Thanks!

**Comment by SilverAzide** [comment]
Yes, this is a quirk of HWiNFO's new (alternate) scheme that does not used shared memory.  It unfortunately writes the stats data to a registry key that you can't see if you are not an admin or using an admin account.  The Rainmeter devs, myself, and a few others chatted with Martin (the HWiNFO developer), but the final decision was to leave it as is.
